carrentalbd2/carrentalbd2/settings.py

```diff
@@ -78,7 +78,14 @@
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
 DATABASES = {
-    "default": {"ENGINE": "django.db.backends.sqlite3", "NAME": BASE_DIR / "db.sqlite3"}
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": "postgres_db",
+        "USER": "admin1",
+        "PASSWORD": "haslo1",
+        "HOST": "localhost",
```

Comment:
In the future maybe we can set these values as environmental variables, so they aren't hardcoded.

Reply:
for now this is ok